services-core/src/price_finding/price_finder_interface.rs

```diff
@@ -99,10 +99,8 @@ pub fn execute_open_solver(
     input_file: &str,
     min_avg_fee_per_order: u128,
 ) -> Result<Output> {
-    let mut command = Command::new("python");
+    let mut command = Command::new("gp_match");
     let open_solver_command = command
-        .current_dir("/app/open_solver")
-        .args(&["-m", "src.match"])
```

Comment:
This is great! So we no longer hard code `app/open_solver`! Amazing! is `gp_match` the name of the pip package?

Reply:
It's the name of a script that gets installed system wide when the open solver is installed via pip.